conf/app.routes

```diff
@@ -38,10 +38,10 @@ PUT           /formBundles/:userId/:formTemplateId/:accessCode/:status/forceStat
 GET           /formtemplates                                                                            uk.gov.hmrc.gform.formtemplate.FormTemplatesController.all
 POST          /formtemplates                                                                            uk.gov.hmrc.gform.formtemplate.FormTemplatesController.upsert
 GET           /formtemplates/get-titles-with-pii/:formTemplateRawId                                     uk.gov.hmrc.gform.formtemplate.FormTemplatesController.getTitlesWithPII(formTemplateRawId: FormTemplateRawId, filters: Option[String], includeJson: Boolean)
-GET           /formtemplates/:formTemplateId                                                            uk.gov.hmrc.gform.formtemplate.FormTemplatesController.get(formTemplateId: FormTemplateId)
+GET           /formtemplates/:formTemplateId/internal                                                   uk.gov.hmrc.gform.formtemplate.FormTemplatesController.get(formTemplateId: FormTemplateId)
 GET           /formtemplates/:formTemplateId/latest                                                     uk.gov.hmrc.gform.formtemplate.FormTemplatesController.getLatest(formTemplateId: FormTemplateId)
 GET           /formtemplates/:formTemplateId/raw/sensitive                                              uk.gov.hmrc.gform.formtemplate.FormTemplatesController.getRawSensitive(formTemplateId: String)
```

Comment:
I think `/raw/sensitive` needs to be migrated also. Doing so would also need change in eeitt-admin

Reply:
Done